beacon-chain/orchestrator/BUILD.bazel

```diff
@@ -2,7 +2,7 @@ load("@prysm//tools/go:def.bzl", "go_library")
 load("@io_bazel_rules_go//go:def.bzl", "go_test")
 
 go_library(
-    name = "orchestrator",
+    name = "go_default_library",
```

Comment:
This is because of `prysm` project convention? (they are using `dafault` naming in this project)

Reply:
Yes. This is a module and all module should have same conventional name.